packages/core/src/Stores/client-store.js

```diff
@@ -2244,6 +2251,20 @@ export default class ClientStore extends BaseStore {
             this.is_populating_account_list = true;
             const authorize_response = await BinarySocket.authorize(is_client_logging_in);
 
+            if (is_social_signup_provider) {
+                const { get_account_status } = await WS.authorized.getAccountStatus();
+
+                const social_identity_provider = get_account_status?.social_identity_provider;
+
+                Analytics.trackEvent('ce_virtual_signup_form', {
+                    action: 'signup_continued',
+                    signup_provider: social_identity_provider,
```

Comment:
can we set it directly?
```suggestion
                    signup_provider: get_account_status?.social_identity_provider,
```